app/controllers/search_controller.rb

```diff
@@ -1,9 +1,18 @@
 class SearchController < ApplicationController
   before_action :require_user
   def index
-    conn = Faraday.get("https://damp-forest-93176.herokuapp.com/images/#{params[:search]}")
-    json = JSON.parse(conn.body, symbolize_names: true)
-    @image_options = ImageOptions.new(json)
+    search = params[:search].split.join('+')
+    if search.match('[a-zA-Z]+')
+      conn = Faraday.get("https://damp-forest-93176.herokuapp.com/images/#{search}")
+      if conn.status == 500
+        redirect_to '/500'
+      elsif conn.status == 200
+        json = JSON.parse(conn.body, symbolize_names: true)
+        @image_options = ImageOptions.new(json)
+      end
+    else
+      redirect_to '/400'
```

Comment:
This is happening when we search gibberish or special characters that the API does not enjoy.

Reply:
I think this method could be refactored out if we end up having time to work on that.